aws/src/integration/java/org/apache/iceberg/aws/glue/TestGlueCatalogTable.java

```diff
@@ -132,6 +133,29 @@ public void testCreateTableBadName() {
                 TableIdentifier.of(namespace, "table-1"), schema, partitionSpec));
   }
 
+  @Test
+  public void testCreateAndLoadTableWithoutWarehouseLocation() {
+    GlueCatalog glueCatalogWithoutWarehouse = new GlueCatalog();
+    glueCatalogWithoutWarehouse.initialize(
+        catalogName,
+        null,
+        new AwsProperties(),
+        glue,
+        LockManagers.defaultLockManager(),
+        new S3FileIO(clientFactory::s3),
+        ImmutableMap.of());
+    String namespace = createNamespace();
+    String tableName = getRandomName();
+    TableIdentifier identifier = TableIdentifier.of(namespace, tableName);
+    try {
```

Comment:
nit: is the try-catch really needed when we expect the test to succeed? Or is it more of a "give an additional hint when it fails"?

Reply:
yes, just try to make the failure reason more explicit

Reply:
thanks, that makes sense in this case